Mathlib/Topology/ContinuousFunction/Bounded.lean

```diff
@@ -1626,4 +1626,26 @@ lemma norm_sub_nonneg (f : α →ᵇ ℝ) :
 
 end
 
+variable {𝕜 : Type*} [NormedField 𝕜] [TopologicalSpace α] [NormedRing γ]
+
+/-- The subtype of compactly supported functions as an ideal. -/
+def CompactlySupportedBoundedContinuousFunction : Ideal (α →ᵇ γ) where
```

Comment:
Can you implement this as a `RingCon` instead of an `Ideal` (which is only a left-ideal)? We're planning to use that for two-sided ideals.

Reply:
sorry, I know this is not specific to mathlib, but I need to sync the branch to `master` because `RingCon` has been separated. How can I do that? I tried to rebase the branch to locally but cannot push it. If I merged master to this branch, there would be thousands of commits made in the meantime...

Reply:
Just merge master.

Reply:
ok, done